runtime/driver/src/main/java/edu/snu/nemo/driver/NemoDriver.java

```diff
@@ -153,13 +153,7 @@ private void startSchedulingUserApplication() {
   public final class FailedEvaluatorHandler implements EventHandler<FailedEvaluator> {
     @Override
     public void onNext(final FailedEvaluator failedEvaluator) {
-      // The list size is 0 if the evaluator failed before an executor started. For now, the size is 1 otherwise.
-      failedEvaluator.getFailedContextList().forEach(failedContext -> {
```

Comment:
Is it necessary to move this code(getteing failed context list) to `RuntimeMaster`? Since handling `FailedEvaluator` belongs to REEF evaluator part. 

If there are any specific reason, could you explain it in comments at `RuntimeMaster`?

Reply:
Unfortunately there's no clear boundary between NemoDriver and RuntimeMaster at the moment regarding using REEF constructs. I've filed an issue to isolate the REEF dependencies within /runtime/driver: https://issues.apache.org/jira/browse/NEMO-93